src/Compilers/CSharp/Portable/xlf/CSharpResources.cs.xlf

```diff
@@ -8602,7 +8602,7 @@ Pokud chcete odstranit toto varování, můžete místo toho použít /reference
       </trans-unit>
       <trans-unit id="ERR_InvalidHashAlgorithmName">
         <source>Invalid hash algorithm name: '{0}'</source>
-        <target state="new">Invalid hash algorithm name: '{0}'</target>
+        <target state="translated">Neplatný název algoritmu hash: {0}</target>
```

Comment:
wrong, should be "Neplatný název hashovacího algoritmu"

Reply:
Hi,
"hash algorithm" has approved terminology "algoritmus hash" and not "hashovacího algoritmu".

The change here would cause inconsistency issues within other products.

Please let me know if your suggested translation is high priority for you or just preferential change.

Thank you,
Jan

Reply:
It's definitely not high priority for me as I don't even use the localized version, but the "approved terminology" is clearly wrong then, as would seem obvious to someone both familiar with the language and knowing that the "hash" in "hash algorithm" is an adjective, not a name of the algorithm.

Reply:
Thank you,
 we will be working on the change of this term based on your feedback.